lightkurve/lightcurve.py

```diff
@@ -1919,6 +1919,66 @@ def plot_river(self, period, epoch_time=None, ax=None, bin_points=1,
             ax.set_aspect(a/b)
         return ax
 
+    def create_transit_mask(self, period, transit_time, duration):
+        """Returns a np.array that is False during transits and True everywhere else.
+
+        This function is flexible to multi-planet systems, allowing users to pass
+        in transit parameters as arrays or lists.
+
+        Parameters
+        ----------
+        period : float, or array-like
+            Period of the transits.
+        duration : float, or array-like
+            Duration of the transits.
+        transit_time : float, or array-like
+            Transit midpoint of the transits.
+
+        Returns
+        -------
+        transit_mask : np.array of Bool
+            Mask that removes transits. Mask is True where there are no transits.
+        """
+
+        # Make sure all params have the same number of entries
+        n_planets = len(np.atleast_1d(period))
+        if any(len(param) != n_planets for param in [np.atleast_1d(transit_time),
+                                                     np.atleast_1d(duration)]):
+            raise ValueError("period, duration, and transit_time must have "
+                             "the same number of values.")
+
+        # Create empty cadence mask
+        transit_mask = np.empty(len(self), dtype=bool)
+        transit_mask[:] = True
+
+        # Create a transit mask for a single planet
+        if isinstance(period, (float, int)):
+            in_transit = self._get_masked_cadences(period, duration, transit_time)
+
+        # Create a multi-planet transit mask
+        elif isinstance(period, (list, np.ndarray)):
+            # Create mask for first planet
+            in_transit = self._get_masked_cadences(period[0], duration[0],
+                                                   transit_time[0])
+            # Iterate through other planets
+            for i,p in enumerate(period[1:]):
+                n = i+1
+                new_in_transit = self._get_masked_cadences(p, duration[n],
+                                                           transit_time[n])
+                in_transit = np.logical_or(in_transit, new_in_transit)
```

Comment:
Lies 1960-1968 seem more complicated than need be.  I think they can be replaced by:
```python
for per, dur, tt in zip(period, duration, transit_time):
    in_transit |= self._get_masked_cadences(period, duration, transit_time)
```